src/gui/sdlmain.cpp

```diff
@@ -381,6 +381,7 @@ static SDL_Rect calc_viewport(int width, int height);
 
 static void CleanupSDLResources();
 static void HandleVideoResize(int width, int height);
+const char *SetProp(std::vector<std::string> &pvars);
```

Comment:
Is there a header this can go somewhere?

Currently it looks like there's no obvious connection about where this function is defined (in programs.cpp)

Reply:
I put the function header in setup.h so that this one is now removed. The SetProp function body is now in setup.cpp.